nginx.conf.template

```diff
@@ -1,51 +1,76 @@
-# /etc/nginx/nginx.conf.template
-
+# /etc/nginx/nginx.conf
 user nginx;
```

Comment:
nginx will throw a warning about user, but it doesn't matter. You can comment out or delete this line

Reply:
This was an oversite deleted on my nginx, but not updated on the template. Thank you.